package/libs/libxml2/Makefile

```diff
@@ -161,8 +161,8 @@ define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
 	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libxml-2.0.pc $(1)/usr/lib/pkgconfig/
 
-	$(INSTALL_DIR) $(2)/share/aclocal/
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/share/aclocal/* $(2)/share/aclocal
+	$(INSTALL_DIR) $(1)/usr/share/aclocal
+	$(CP) $(PKG_BUILD_DIR)/aclocal.m4 $(1)/usr/share/aclocal
```

Comment:
why do you install this into the normal staging dir and not the host staging dir now?